scripts/cibuild

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GT_SITE_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0") COMMAND OPTION[S]
+Run tests
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+        exit 0
+    fi
+        # shellcheck disable=SC2086
```

Comment:
Shellcheck was throwing an error when the whole path was quoted. In [99b6ad9](https://github.com/geotrellis/geotrellis-site-deployment/pull/12/commits/99b6ad98db3d6080f7f0d213706dfaa3aa085e12) I just quoted `${DIR}`